1.download-data/scripts/download_data.py

```diff
@@ -0,0 +1,30 @@
+import urllib.request
+import shutil
+import os
+
+
+def downloadData(
+    locationData: str = "temp.zip",
+    url: str = r"https://ndownloader.figshare.com/files/18501824?private_link=f41918598b1ff5116825",
+):
+    """
+    locationData: The name of the zip file. 
+    url: This is the download link. Pasting this link in your browser should tricker a download.
+    """
+    # Check if path goes to zipfile
+    if not locationData.endswith(".zip"):
+        locationData = os.path.join(locationData, "fullData.zip")
+
+    # Check if file already exist
+    if os.path.exists(locationData):
+        print("file already exists. Nothing is downloaded")
```

Comment:
This is great! Maybe also want to do a quick md5sum check if the data exists. Perhaps something like:

```python
import hashlib

if os.path.exists(locationData):
    with open(file_name, "rb") as fh:
        md5 = hashlib.md5()
        data = fh.read()
        md5.update(data)
        md5sum = md5.hexdigest()
    assert md5sum == "6583e93eaee9e1cec8dfb11149099174"
```

Reply:
This avoids the partial download scenario

Reply:
This looks intresting. I am going to look it to it